### PR TITLE
11 Hospital Readmission

### PR DESCRIPTION
**Pull request by JCardenasRdz**
Finished Project 11

### CODE REVIEW

File: Projects/11_EDA_Reduce_Hospital_Readmissions/11_Readmission.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+import plotly.plotly as py
+
+def load_and_clean():
+    column_names = ['Number of Discharges', 'Excess Readmission Ratio', 'Number of Readmissions', 'State']
+    df = hospital_read_df = pd.read_csv('data/cms_hospital_readmissions.csv', skip_blank_lines = True)
+    df = df[column_names]
+    df.columns=['Discharges', 'Excess_Ratio', 'Readmissions', 'State']
+    df = df.dropna()
+    df.Discharges = df.Discharges.astype(float)
+    df = df.iloc[np.isreal(df.Discharges),:]
```
Comment (amirziai) [inline Projects/11_EDA_Reduce_Hospital_Readmissions/11_Readmission.py:15]: Are complex types possible in `Discharges`?

File: Projects/11_EDA_Reduce_Hospital_Readmissions/11_Readmission.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    plt.figure(figsize=(12,7))
+    plt.subplot(1,3,1);
+    plt.scatter(df.Discharges.apply(np.log10), df.Readmissions.apply(np.log10), c=df.Excess_Readmission_Ratio)
+    plt.colorbar()
+    plt.xlabel('Log 10 of Discharges')
+    plt.ylabel('Log 10 of Readmissions')
+    plt.title('Excess Readmission Ratio')
+
+    plt.subplot(1,3,2);
+    bool_ = df.Excess_Readmission_Ratio < 1.0
```
Comment (amirziai) [inline Projects/11_EDA_Reduce_Hospital_Readmissions/11_Readmission.py:28]: In my experience using descriptive variable names makes your code much more legible, specially as the project size grows. I would've used `excess_readmission_ratio_great_than_one`, it is very long but the smaller cognitive overhead outweighs the variable length in my opinion.

File: Projects/11_EDA_Reduce_Hospital_Readmissions/11_Readmission.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    plt.scatter(df.Discharges.apply(np.log10)[bool_], df.Readmissions.apply(np.log10)[bool_], c=df.Excess_Readmission_Ratio[bool_])
```
Comment (amirziai) [inline Projects/11_EDA_Reduce_Hospital_Readmissions/11_Readmission.py:29]: You are calculating log for a number of rows that you'll later filter out. Can you do the filtering earlier?

**Review by amirziai — APPROVED** [review]
Great job!

Left performance and stylistic comments.

File: Projects/11_EDA_Reduce_Hospital_Readmissions/11_Readmission.py (hunk 2 above)
Comment (JCardenasRdz) [reply Projects/11_EDA_Reduce_Hospital_Readmissions/11_Readmission.py:28]: go it.

File: Projects/11_EDA_Reduce_Hospital_Readmissions/11_Readmission.py (hunk 3 above)
Comment (JCardenasRdz) [reply Projects/11_EDA_Reduce_Hospital_Readmissions/11_Readmission.py:29]: I forgot to delete ```myscatter```. I did not use this function in the final version.

**Pull request merged**